src/interfaces/propsInterfaces.ts

```diff
@@ -11,3 +12,33 @@ export interface EditEventProps {
 export interface EventProps {
   event: Event;
 }
+
```

Comment:
Instead of calling this file `propsInterfaces` add the related interfaces within their own file. For example, any user-related interface should be put within the `UserInterfaces.ts` file, and so on for other interfaces.